Cargo.toml

```diff
@@ -21,6 +21,8 @@ log = "0.4"
 serde_json = "1.0"
 thiserror = "2.0.11"
 tokio = { version = "1", features = ["full"] }
+payjoin = { version = "0.23.0", features = ["v1", "v2", "io"] }
```

Comment:
I don't believe you need the `v1` feature, you want to use the backwards compatible `v2` feature to handle `v1` rather than the exclusive `v1` feature.